### PR TITLE
Shim this.import for Engines support

### LINKED ISSUE #523: Doesn't work in engines
I think @nathanhammond said that `this.import` needs to be used otherwise the css doesn't get added and the build errors.

### PR DESCRIPTION
**Pull request by knownasilya**
Resolves #523

### CODE REVIEW

**Comment by knownasilya** [comment]
Ok, should be ready for merge. Works in Engines now, not really sure how to write a test or if it's even needed.

**Comment by Cryrivers** [comment]
Seems it cannot import Velocity.js

screen shot 2016-11-11 at 10 27 12 am

**Comment by knownasilya** [comment]
Ah, yes will fix

File: index.js [hunk 1]
```diff
@@ -85,34 +107,29 @@ module.exports = {
...
-    if (typeof app.import !== 'function' && app.app) {
-      app = app.app;
-    }
-
     if (process.env.EMBER_CLI_FASTBOOT) {
       // in fastboot we use the shim by itself, which will make
       // importing velocity a noop.
-      app.import('vendor/shims/velocity.js');
+      this.import('vendor/shims/velocity.js');
     } else if (haveShimAMDSupport(app)) {
```
Comment (Cryrivers) [inline index.js]: I believe this should check `this` instead of `app` as well.

**Comment by knownasilya** [comment]
Ok, give it a try again @Cryrivers

**Comment by Cryrivers** [comment]
@knownasilya it works like a charm. thanks!

**Pull request merged**